.github/workflows/run-tests.yml

```diff
@@ -1,6 +1,7 @@
 name: Run Jest Tests
 
-on: [push, pull_request]
+on:
+    pull_request:
```

Comment:
does this work? looks like it might give a YAML syntax error

Reply:
I think you are right. Fixed